dev-system/resources/kms_examples/rucontenttype

```diff
@@ -1,75 +1,99 @@
-"USE SERVICE API","WEB MAP TILE SERVICE (WMTS)","7aac9f91-20c4-4234-9153-e850c8ace8a9"
-"USE SERVICE API","","d117cf5c-8d23-4662-be62-7b883cecb219"
-"VIEW RELATED INFORMATION","ALGORITHM DOCUMENTATION","fcc9411c-a1c9-415d-a16c-75c42f2cec45"
-"VIEW RELATED INFORMATION","ALGORITHM THEORETICAL BASIS DOCUMENT (ATBD)","fd01f7ec-fdf6-4440-b974-75f12fb4ec5f"
-"VIEW RELATED INFORMATION","ANOMALIES","914cbb7e-5b20-4bcd-86e3-ffcfa26f0a73"
-"VIEW RELATED INFORMATION","CASE STUDY","3112d474-b44f-4af1-8266-c3dd6d28220f"
-"VIEW RELATED INFORMATION","DATA CITATION POLICY","40cf5001-15ec-4d9a-913c-bb323f2974fc"
-"VIEW RELATED INFORMATION","DATA QUALITY","0eba3253-8eb7-4e43-9627-9cff48775e27"
-"VIEW RELATED INFORMATION","DATA RECIPE","547600e9-b60a-44eb-b14b-5c6e1f2c094e"
-"VIEW RELATED INFORMATION","DELIVERABLES CHECKLIST","be0460d8-ca8e-45c8-b637-8fb4ce5a5e97"
-"VIEW RELATED INFORMATION","GENERAL DOCUMENTATION","aebf20eb-39c7-4f4f-aecf-a628f703867b"
-"VIEW RELATED INFORMATION","HOW-TO","7ebd73e5-b0aa-4cf2-ace5-1d3890c2c3ce"
-"VIEW RELATED INFORMATION","IMPORTANT NOTICE","2af2cfc4-9390-43da-8fa8-1f272e8ee0b0"
-"VIEW RELATED INFORMATION","INSTRUMENT/SENSOR CALIBRATION DOCUMENTATION","fc3c1abb-92c1-49c2-90d4-161c70cff44a"
-"VIEW RELATED INFORMATION","MICRO ARTICLE","4f3c0b04-1fe6-4e11-994a-9cc4afd09ce0"
-"VIEW RELATED INFORMATION","PI DOCUMENTATION","367f8b8a-e57e-4c49-b971-0b5c6a484186"
-"VIEW RELATED INFORMATION","PROCESSING HISTORY","7cfa5214-7f69-4355-b259-286be88f25d1"
-"VIEW RELATED INFORMATION","PRODUCT HISTORY","b292f51f-d2b4-4e65-84a9-e50306238989"
-"VIEW RELATED INFORMATION","PRODUCT QUALITY ASSESSMENT","b7ed88ce-3f04-40ea-863e-ac58bd048ff3"
-"VIEW RELATED INFORMATION","PRODUCT USAGE","1132a0fc-888b-4332-ad0a-dc5c6e615afa"
-"VIEW RELATED INFORMATION","PRODUCTION HISTORY","0b597285-eaac-4cbd-94cc-d87ae8046681"
-"VIEW RELATED INFORMATION","PUBLICATIONS","13a4deec-bd22-4864-9804-77fac181f484"
-"VIEW RELATED INFORMATION","READ-ME","aa3cea98-b20a-4de8-8f22-7a8b30784625"
-"VIEW RELATED INFORMATION","REQUIREMENTS AND DESIGN","86b8b121-d710-4c5b-84b0-7b40717f6c76"
-"VIEW RELATED INFORMATION","SCIENCE DATA PRODUCT SOFTWARE DOCUMENTATION","e8e6e972-832f-4501-a721-4108f33332d6"
-"VIEW RELATED INFORMATION","SCIENCE DATA PRODUCT VALIDATION","15b0a4c4-b39d-48f5-92d2-905e45e6dc6a"
-"VIEW RELATED INFORMATION","USER FEEDBACK PAGE","ab2fce71-e5f9-4ba6-bfb1-bc428a8b7dd8"
-"VIEW RELATED INFORMATION","USER'S GUIDE","d1996d91-e824-4b24-b94e-3aae4543b63b"
-"VIEW RELATED INFORMATION","","5ec1bb9d-0efc-4099-9b31-ec791bbd8145"
+"Hits: 98","page_num: 1","page_size: 2000","Keyword Version: draft","Revision: 2021-07-30 10:51:26","Timestamp: 2021-07-30 10:58:11","Terms Of Use: https://cdn.earthdata.nasa.gov/conduit/upload/5182/KeywordsCommunityGuide_Baseline_v1_SIGNED_FINAL.pdf","The most up to date XML representations can be found here: https://gcmd.sit.earthdata.nasa.gov/kms/concepts/concept_scheme/rucontenttype/?format=xml""Case native"
```

Comment:
Get rid of "Hits: 98","page_num: 1","page_size: 2000",

Reply:
KMS does this now for schemes which have not been "grandfathered", which is all the short lists. I feel more comfortable knowing that CMR can handle (and ignore) this new content. Also, if there are more then 2000 related urls, then KMS will page them, CMR does not support this yet, but it is something we should look to fix.